### PR TITLE
Added Huaweicloud hwc_smn_topic module

### PR DESCRIPTION
**Pull request by zhongjun2**
SUMMARY
Add a new modules  hwc_smn_topic that can fetch huaweicloud Simple Message Notification.

COMPONENT NAME
lib/ansible/modules/cloud/huawei/hwc_smn_topic.py

ANSIBLE VERSION
ansible 2.8

### CODE REVIEW

**Comment by niuzhenguo** [comment]
shipit

File: lib/ansible/modules/cloud/huawei/hwc_smn_topic.py [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+      password: "{{ password }}"
+      domain_name: "{{ domain_name }}"
+      project_name: "{{ project_name }}"
+      region: "{{ region }}"
+      name: "ansible_smn_topic_test"
+      state: present
+'''
+
+RETURN = '''
+    create_time:
```
Comment (resmo) [inline lib/ansible/modules/cloud/huawei/hwc_smn_topic.py]: the additional indentation seems unnecessary, would you mind to streamline it?

Comment (zhongjun2) [reply lib/ansible/modules/cloud/huawei/hwc_smn_topic.py]: Thanks!  Done  and added integration tests.

**Comment by TommyLike** [comment]
Tested as below:

WechatIMG70

**Comment by TommyLike** [comment]
shipit

**Comment by edisonxiang** [comment]
shipit

**Comment by zhongjun2** [comment]
bot_status

**Pull request closed without merging**